### PR TITLE
Ember 1.13

### PR DESCRIPTION
**Pull request by jaswilli**
Update to Ember 1.13.2.  The highlights:
- No more views.  All views have been converted to components or refactored away.
- ArrayController/ItemController usage has been converted to regular controllers and components.
- Deprecated Ember.Select no longer used, replaced with a new component (gh-select-native).

### CODE REVIEW

**Comment by halfdan** [comment]
@jaswilli Epic job! :+1:

File: core/client/app/components/gh-post-tags-input.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    //     mouseDown: function (event) {
+    //         event.preventDefault();
+    //     },
+
+    //     mouseUp: function (event) {
+    //         event.preventDefault();
+    //         this.get('parentView.controller').send('addTag',
+    //             this.get('suggestion.tag'));
+    //     }
+    // }),
```
Comment (halfdan) [inline core/client/app/components/gh-post-tags-input.js]: Remove?

File: core/client/app/controllers/application.js [hunk 2]
```diff
@@ -25,23 +25,23 @@ export default Ember.Controller.extend({
     }),
 
     actions: {
-        topNotificationChange (count) {
+        topNotificationChange: function (count) {
             this.set('topNotificationCount', count);
         },
 
-        toggleAutoNav () {
+        toggleAutoNav: function () {
```
Comment (halfdan) [inline core/client/app/controllers/application.js]: Any reason for switching this back to the verbose syntax?

File: core/client/app/components/gh-post-tags-input.js (hunk 1 above)
Comment (jaswilli) [inline core/client/app/components/gh-post-tags-input.js]: Eventually (soon).  The tag input widget is going to be reimplemented, so I did enough to clear the deprecation warnings on the existing implementation and then left it there as a reference.

File: core/client/app/controllers/application.js (hunk 2 above)
Comment (jaswilli) [inline core/client/app/controllers/application.js]: I don't have any issue with the shortcut syntax but:

1. That goes against our existing style (no space between function name and parentheses), but the linter isn't yet smart enough to handle that particular bit of ES6 sugar.
2. We haven't had any actual discussion about what parts of ES6 we're going to use, what the style preferences will be and how that relates to the backend where we're still (realistically) stuck with ES5.

So for now I just reverted the few instances of these types of declarations.  I'm not asserting any sort of opinion aside from wanting to keep things consistent.

**Comment by jaswilli** [comment]
Commits squashed.

**Comment by halfdan** [comment]
Squeeeee! Merge this magic beast :)

File: core/client/app/controllers/application.js (hunk 2 above)
Comment (ErisDS) [inline core/client/app/controllers/application.js]: Related: http://nodejsreactions.tumblr.com/post/99325652420/discussing-the-merits-of-new-es6-features

**Comment by ErisDS** [comment]
I have taken this for a spin and it seems a-ok. Not sure what caused the merge conflict, much sorry if I merged something I shouldn't. If you update will ensure this goes in next.

**Comment by jaswilli** [comment]
Rebased.

**Pull request merged**